### PR TITLE
RHIDP-3515: Move authentication related content to a new category (#435)

### PR DESCRIPTION
**Pull request by themr0c**
Manual backport as the automated cherry-pick failed on a conflict.

**Version(s):**

**Issue:**

**Link to docs preview:**

**Reviews:**
- [ ] SME: @ mention assignee
- [ ] QE: @ mention assignee
- [ ] Docs review: @ mention assignee

**Additional information:**

### CODE REVIEW

**Comment by nickboldt** [comment]
1.3 doc PR was https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/435 for RHIDP-3515

File: assemblies/assembly-configuring-authorization-in-rhdh.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
+[id='configuring-authorization-in-rhdh']
+= Configuring authorization in {product}
+
+include::modules/admin/con-rbac-overview.adoc[leveloffset=+1]
+
+
+include::modules/admin/ref-rbac-permission-policies.adoc[leveloffset=+1]
```
Comment (hmanwani-rh) [inline assemblies/assembly-configuring-authorization-in-rhdh.adoc:7]: @themr0c Permission policies are a component of the RBAC feature. Currently, it seems that permission policies and RBAC are presented as two distinct options for configuring authorization. This hierarchy appears to be suitable for now, given that we only have the RBAC feature. However, as we introduce additional authorization options in the future, we may need to reconsider and revise this structure. what do you think?

**Review by hmanwani-rh — COMMENTED** [review]
@themr0c I have a question regarding the RBAC structure, which I’ve included below. I wasn’t able to raise this question in the original PR as it was merged before the peer review. If I recall correctly, we agreed that restructuring requires approval from two peer reviewers, correct?

File: assemblies/assembly-configuring-authorization-in-rhdh.adoc (hunk 1 above)
Comment (themr0c) [reply assemblies/assembly-configuring-authorization-in-rhdh.adoc:7]: Agree. It was out of scope of this PR.

**Pull request merged**